source/fundamentals/crud/write-operations/insert.txt

```diff
@@ -4,9 +4,194 @@ Insert a Document
 
 .. default-domain:: mongodb
 
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+   
 Overview
 --------
 
-In this section, we show you how to call the write operations to **insert**
-documents from a collection in your MongoDB database.
+In this guide, you can learn how to **insert** documents into a MongoDB
+collection.
 
+Before you can retrieve, update, or delete information such as user
+profiles or orders in MongoDB, you first need to perform an insert
+operation to load data. An insert operation inserts one or more
+documents into a MongoDB collection using one of the following methods:
+``InsertOne()``, ``InsertMany()``, or ``BulkWrite()``.
+
+The following sections focus on ``InsertOne()`` and ``InsertMany()``. For an
+example on how to use the ``BulkWrite()`` method, see the :doc:`Bulk
+Operations Usage Example </usage-examples/bulkWrite>`.
+
+A Note About ``_id``
+~~~~~~~~~~~~~~~~~~~~
+
+When inserting a document, MongoDB enforces one constraint on your
+documents by default. Each document *must* contain a unique ``_id``
```

Comment:
```suggestion
documents by default: each document *must* contain a unique ``_id``
```

Suggestion: As two separate sentences, the first sentence seems to be incomplete